duckdb_engine/tests/test_reflection.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy import MetaData, create_engine, text
+
+engine = create_engine("duckdb:///:memory:")
```

Comment:
1. you need to put this inside a test function for pytest to pick it up.
2. I would probably put this inside test_basic.py instead of making a whole new file.

so, would look something like

```
# test_basic.py
# other tests up here...

def test_reflection(engine: Engine) -> None:
    from sqlalchemy import MetaData

    metadata = MetaData()

    with engine.connect() as conn:
        conn.execute(text("CREATE TABLE tbl(col1 INTEGER)"))
        conn.commit()
    metadata.reflect(engine)
```

and then you would run this with something like `poetry run pytest -k reflection duckdb_engine/tests/test_basic.py`

Reply:
Sounds good, I'll make the update. Thanks for the tip!

Reply:
Done in the latest commit!